QUEST.tsv

```diff
@@ -13,15 +13,15 @@ QUEST_20150317_000012	Bokor Master
 QUEST_20150317_000013	So, you want to learn more about this stone slate. {nl}If you wish, I shall show you the will of its creator.
 QUEST_20150317_000014	Goddess Laima
 QUEST_20150317_000015	By the time you get this message, much time will have passed. {nl}I am the Goddess of Fate and Wisdom, Laima.
-QUEST_20150317_000016	600 years ago, I foresaw a lot of grief and death. {nl}If you're seeing this message, then I'm afraid all I've foreseen has likely come to pass.
-QUEST_20150317_000017	The demons have started the first of the cataclysms, and all the Goddesses have vanished. {nl}I knew this would happen and did all I could to change this fate, but, my power alone was not enough.
+QUEST_20150317_000016	600 years ago, I foresaw a great deal of death and suffering. {nl}If you're seeing this, then I'm afraid what I foresaw has come to pass.
+QUEST_20150317_000017	The demons have started the first of the cataclysms, and all of the goddesses have vanished. {nl}I knew this would occur and did everything I could to prevent this fate, but my power was not enough.
 QUEST_20150317_000018	Instead, I wandered through many far-seeing dreams for an answer. {nl}You were at the end of them… a Savior.
-QUEST_20150317_000019	However, knowing of my clairvoyance, the demons would never rest until they saw me gone. {nl}I was forced to hide away, until the time of your appearance.
-QUEST_20150317_000020	I resolved to divide my being and subsist inside these many messages. {nl}I continued to secretly create them, even as the kingdoms flew by over the millennia.
-QUEST_20150317_000021	Savior, these messages are scattered across axes of time. {nl}Once they're all brought together, I will regain my complete form and help against the demons.
-QUEST_20150317_000022	This divine task may be a heavy burden, but it is something you must bear. {nl}Perhaps, in the muddy midst of your trials and tribulations, something may come about.
-QUEST_20150317_000023	Nevertheless, only you and I can save this world.
-QUEST_20150317_000024	I've passed the next Revelation to the first Paladin. {nl}I'll be waiting for you at the highest flower garden.
+QUEST_20150317_000019	However, knowing of my clairvoyance, the demons would not ignore what I had seen. {nl}I was forced to hide, until the time of your appearance.
+QUEST_20150317_000020	I decided to divide my being inside of many messages. {nl}Messages that I continued to create secretly for many millennia.
+QUEST_20150317_000021	Savior, if you will help me regain my form we can stop the demons sinister plans.{nl}You must collect the revelations that I have hidden throughout many axis of time.
```

Comment:
I'm still not pleased to use the word 'collect', it's a word that you'll see all the time in other quests. It almost makes the word sound trivial.

You could say:
> You must recover the revelations that I have hidden throughout many axis of time.